### ISSUE TITLE
Detect outliers in the standby consumption

### ISSUE THREAD

**Issue by JrtPec** [opengridcc/opengrid#148]
We can easily develop an algorithm to detect these kind of outliers (I'm thinking a ransac regressor)
![standby_vertical_a7d523e79909ad098bccc337c5da84a0](https://cloud.githubusercontent.com/assets/4570408/19213179/88a324a8-8d64-11e6-8010-e4d4d892a49d.png)

**Comment by GMathyssen** [comment]
Can I use the scikit-learn library for it?

**Comment by JrtPec** [comment]
Yes! I was thinking about a [RANSAC-algorithm](http://scikit-learn.org/stable/auto_examples/linear_model/plot_ransac.html#example-linear-model-plot-ransac-py).

I have implemented it before, so I might have some code snippets that could be used to start.

**Comment by JrtPec** [comment]
This should get you started!

```python
import numpy as np
from sklearn import linear_model

x = []  # list with x-values
y = []  # list with y-values
# make sure that x and y have the same size!

model_ransac = linear_model.RANSACRegressor(linear_model.LinearRegression())
model_ransac.fit(x, y)

# use inlier_mask_ to get a boolean series of the inliers
inliers = model_ransac.inlier_mask_
outliers = np.logical_not(inliers)
```

**Comment by Ryton** [comment]
@saroele  JrtPec: Maybe also kick out FL03001550 from the analysis? This seriously skews the analysis/mean. I'm convinced this is not a residential building (an office?) as it has a night/weekend consumption of 5kW and week-day consumption of 15-20kW.

https://opengrid.be/sensor/565de0a7dc64d8370aa321491217b85f

**Comment by saroele** [comment]
yes, this is an office :-)

With a basic categorisation approach we should be able to select only
dwellings or only offices for a specific analysis.  That´s the idea of the
SAREF or Haystack implementations (see also #116).

**Comment by JrtPec** [comment]
It wouldn't be too hard to add a "building type" argument to the sites in the houseprint, both in the GDocs file as in the code + parser. I would like to see what naming conventions SAREF uses, but other than that it's pretty straightforward.
